next.json

```diff
@@ -921,8 +921,8 @@
             {"value": 2, "name": "transfer dst"},
             {"value": 4, "name": "sampled"},
             {"value": 8, "name": "storage"},
-            {"value": 16, "name": "color attachment"},
-            {"value": 32, "name": "depth stencil attachment"}
+            {"value": 16, "name": "output attachment"},
+            {"value": 32, "name": "input attachment"}
```

Comment:
Maybe we could skip adding input attachments for now since we are not going to implement them for a while.